src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
   // threads since last reset
   double time_system = 3;
+
+  // Number of requests received
+  uint64 reqs_received = 4;
+
+  // Number of responses sent
+  uint64 responses_sent = 5;
 }
 
 // Histogram params based on grpc/support/histogram.c
```

Comment:
My feeling is that micros won't necessarily work well with the exponential formula we use to compute the bucket sizes.